src/aequitas/plotting.py

```diff
@@ -0,0 +1,1064 @@
+        """
+        Finds relative index (row) of reference group value for a given metric.
+
+        :param disparities_table: A disparity table. Output of bias.get_disparity or
+            fairness.get_fairness functions
+        :param attribute_name: The attribute to plot metric against. Must be a column
+            in the disparities_table.
+        :param group_metric: The metric to plot. Must be a column in the
+            disparities_table.
+        :param ref_group_flag: string indicating column indicates reference group
+            flag value. Default is '_ref_group_value'.
+        :param model_id: Model ID number. Default is 1.
+
+        :return: Integer indicating relative index of reference group value row.
+        """
+        # get absolute metric name from passed group metric (vs. a disparity name)
+        abs_metric = "".join(group_metric.split('_disparity'))
+
+        all_ref_groups = self.__assemble_ref_groups(disparities_table, ref_group_flag)
+        ref_group_name = all_ref_groups[attribute_name][abs_metric]
+
+        # get index for row associated with reference group for that model
+        ind = list(disparities_table.loc[(disparities_table['attribute_name'] == attribute_name) &
+                                     (disparities_table['attribute_value'] == ref_group_name) &
+                                     (disparities_table['model_id'] == model_id)].index)
+
+        # there should only ever be one item in list, but JIC, select first
+        idx = ind[0]
+
+        relative_ind = disparities_table.index.get_loc(idx)
+        return relative_ind, ref_group_name
+
+
+    def plot_group_metric(self, group_table, group_metric, ax=None, ax_lim=None,
+                          title=True, label_dict=None,
+                          min_group_size = None):
+        '''
+        Plot a single group metric's absolute metrics
+
+        :param group_table: A group table
+        :param group_metric: The metric to plot. Must be a column in the group_table
+        :param ax: A matplotlib Axis. If not passed a new figure will be created.
+        :param ax_lim: Maximum value on x-axis, used to match axes across subplots
+            when plotting multiple metrics. Default is None.
+        :param title: Whether a title should be added to the plot. Default is True.
+        :param label_dict: Optional dictionary of replacement labels for data.
+            Default is None.
+        :param min_group_size: Minimum size for groups to include in visualization
+            (as a proportion of total sample)
+
+        :return: matplotlib.Axis
+        '''
+        assert (group_metric in group_table.columns), \
+            f"Specified disparity metric '{group_metric}' not in 'group_table'."
+
+        if any(group_table[group_metric].isnull()):
+            raise IOError(f"Cannot plot {group_metric}, has NaN values.")
+
+        if ax is None:
+            fig, ax = plt.subplots(figsize=(10, 5))
```

Comment:
it looks like `fig` is unused. you can indicate this to others with a leading underscore:

    (_fig, ax) = ...